Mathlib/Combinatorics/SimpleGraph/Connectivity/WalkCounting.lean

```diff
@@ -169,6 +171,35 @@ instance : Decidable G.Connected := by
 instance instDecidableMemSupp (c : G.ConnectedComponent) (v : V) : Decidable (v ∈ c.supp) :=
   c.recOn (fun w ↦ decidable_of_iff (G.Reachable v w) $ by simp)
     (fun _ _ _ _ ↦ Subsingleton.elim _ _)
+
+lemma odd_card_odd_components (ho : Odd (Fintype.card V)) :
+    Odd (Nat.card ({(c : ConnectedComponent G) | Odd (Nat.card c.supp)})) := by
```

Comment:
The statement uses both `Fintype.card` and `Nat.card`. Would it be possible to only use one, and consistently use that one in the proof as well?

Reply:
Used `Nat.card` consistently